src/main/scala/is/hail/expr/ir/AggOp.scala

```diff
@@ -9,11 +9,13 @@ import scala.reflect.ClassTag
 
 case class AggSignature(
   op: AggOp,
-  inputType: Type,
   constructorArgs: Seq[Type],
   initOpArgs: Option[Seq[Type]],
-  seqOpArgs: Seq[Type]
-)
+  seqOpArgs: Seq[Type]) {
+  assert(seqOpArgs.nonEmpty)
+
+  def inputType: Type = seqOpArgs.head
```

Comment:
This doesn't appear to be used?

Reply:
It's used in Interpret as arguments to construct the aggregator
```
        val aggType = aggSig.inputType
         ...
         case Collect() => new CollectAggregator(aggType)
          case Counter() => new CounterAggregator(aggType)
          case CollectAsSet() => new CollectSetAggregator(aggType)
```

I debated taking it out of AggSignature and just use `aggSig.seqOpArgs.head` where needed.